### ISSUE TITLE
[FEATURE] Add additional digit of precision for specifying learning rate and other parameters

### ISSUE THREAD

**Issue by tmostak** [h2oai/h2o-llmstudio#661]
### 🚀 Feature

It would be helpful to be able to specify learning rate and other parameters with 7 decimal digits, (i.e. 1e-7), rather than the current 6.

### Motivation

When getting down to final fine-tuning of models, I find that at being able to adjust learning rate in steps of 5e-7 or less can be helpful to get the best results. However, the current UX only allows steps of 1e-6. Currently I make changes to the code itself to allow the exact step I want, but this requires a code change for every model run rather than being able to do this through the UI.

**Comment by pascal-pfeiffer** [comment]
Thanks for raising this issue again. Wave now supports up to 1e-20 as values for the spinbox, so should be straight forward to allow lower values in H2O LLM Studio.

ref: https://github.com/h2oai/wave/issues/1525
